admin/app/components/certification-centers/invitations-action.js

```diff
@@ -22,6 +26,25 @@ export default class CertificationCenterInvitationsAction extends Component {
     ];
   }
 
+  get rolesOptions() {
+    return [
+      {
+        label: this.intl.t('common.roles.admin'),
+        value: 'ADMIN',
+      },
+      { label: this.intl.t('common.roles.member'), value: 'MEMBER' },
+    ];
+  }
+
+  get certificationCenterRoleValue() {
+    return this.invitationRole === 'NULL' ? null : this.invitationRole;
+  }
```

Comment:
Ce `this.invitationRole === 'NULL'` me gêne car je n'ai pas trouvé à quel cas cela correspond. J'ai de mon côté en local modifié
```javascript
return this.invitationRole === 'NULL' ? null : this.invitationRole;
```
en 
```javascript
return this.invitationRole;
```
et l'application continue de fonctionner correctement dans tous les cas que j'ai essayés. Donc je pense qu'on peut se passer de ce _getter_ `certificationCenterRoleValue`. D'autant que le nom de ce getter est gênant car dans le code lorsqu'on manipule ou vérifie un `role` au niveau algorithmique/logique métier, c'est toujours une propriété avec un nom dans le genre `role` qu'on manipule et non un nom dans le genre `roleValue` (sauf pour `rolesOptions` mais là on est dans un _component_ graphique).

Reply:
Cela correspond au faite que tu sélectionnes le mode Automatique, dans le sélecteur de rôle, qui a comme valeur "NULL".